### ISSUE TITLE
Add code coverage badges to the repo

### ISSUE THREAD

**Issue by skodamarthi** [intuit/user-data-for-fraud-prevention#19]

**Comment by christyjacob4** [comment]
Hi @skodamarthi is this something I can take up for hacktoberfest?

**Comment by reubenae** [comment]
@christyjacob4 you are welcome to work on this for hacktoberfest but for it to count you will need to wait until October to start before opening your PR. We will be applying hacktoberfest to relevant issues once October begins. We do not allow contributors to "save" issues for themselves and block others from contributing.

**Comment by christyjacob4** [comment]
Yeap. Makes sense. 👍

**Comment by christyjacob4** [comment]
@reubenae @skodamarthi  I will need some help to include codecov in the repository. 
Adding the badge is straight forward, but we'll first need to add CodeCov to our repo. Let me know when we can do that. 
I found this useful to setup CodeCov
https://github.com/TechnionYP5777/SmartCity-Market/wiki/Integrating-Codecov-with-a-GitHub-project-(Coverage-tool)

**Comment by skodamarthi** [comment]
Hi @christyjacob4 We already added jest-coverage-badges to the package. https://github.com/intuit/user-data-for-fraud-prevention/blob/3a198cab8d27d71dd5d6ec882d4994f6cefb9059/package.json#L9. Idea for this issue is to add the badges generated by jest-coverage-badges to the repo. Does that help?

**Comment by christyjacob4** [comment]
Yeap @skodamarthi got it.. I see that `jest-coverage-badges` can generate 4 different badges. Which one should be displayed in the README ?

```
./coverage/badge-statements.svg
./coverage/badge-lines.svg
./coverage/badge-functions.svg
./coverage/badge-branches.svg

```

**Comment by christyjacob4** [comment]
In order to add the badges to the repo, Im thinking of running `npm run test:badges` in circleci `config.yml`

**Comment by christyjacob4** [comment]
<img width="600" alt="Screenshot 2020-09-30 at 2 14 19 PM" src="https://user-images.githubusercontent.com/20852629/94740309-1193b980-0390-11eb-89e9-410cd7a8091c.png">
Is this what is expected?

**Comment by skodamarthi** [comment]
> <img alt="Screenshot 2020-09-30 at 2 14 19 PM" width="600" src="https://user-images.githubusercontent.com/20852629/94740309-1193b980-0390-11eb-89e9-410cd7a8091c.png">
> 
> Is this what is expected?

Looks good @christyjacob4! But can you please add all the four badges? You can keep circleci, npm and contributions badges in one line and all the coverage badges under a Section titled Code Coverage. 
<img width="660" alt="Screen Shot 2020-09-30 at 22 37 25" src="https://user-images.githubusercontent.com/4538858/94742540-c1a3fb00-036d-11eb-8194-00bf2a92b75d.png">

**Comment by christyjacob4** [comment]
one question.. Since we don't have the ./coverage directory yet , these badges wont render unless we get run `npm run test:badges` from circle ci

**Comment by skodamarthi** [comment]
Updating this issue to only add code coverage badges as this is going to take some time. Will create a separate issue for npm version badge for GHC participants to work on it in parallel.

**Comment by christyjacob4** [comment]
Sure makes sense 👍

**Comment by adithyaakrishna** [comment]
Can I work on this issue if its available, I can add the perfect logos and work on their alignment also 😶
